examples/profile-client-protobufs/build.gradle

```diff
@@ -1,20 +1,4 @@
 apply from: "../../gradle/shared.gradle"
+apply from: "../../gradle/protobuf.gradle"
 
 group = 'com.palantir.atlasdb.examples'
-
-apply plugin: 'com.google.protobuf'
-
-// The protoc version atlasdb uses does not yet support Apple M1 aarch64
-// (see https://github.com/protocolbuffers/protobuf/issues/8062), so use x86_64.
```

Comment:
Have you tested this change on an M1?

Reply:
Yes, running `generateProto` succeeds on an M1.